modules/ipi-install-ipv6-network-requirements.adoc

```diff
@@ -0,0 +1,53 @@
+[discrete]
+== IPv6 considerations
+
+.SLAAC Addressing
+
+If you don't plan to use SLAAC footnote:[Stateless Address AutoConfiguration] addresses on your OpenShift node, then it should be disabled for `baremetal` networks, that means that if your network equipment is configured to send SLAAC addresses when replying to Route Advertisements that behavior should be changed, so it only sends the route and not the SLAAC address.
+
+You can install `ndptool` on your system in order to check what your RAs look like:
+
+[source,shell]
+----
```

Comment:
In our updated style guidelines, we now break up commands and output into separate code blocks: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
> Adding @iranzo

@johnwilkins 

All changes (including the ones from @ahardin-rh ) included in https://github.com/openshift-kni/baremetal-deploy/pull/547 except for the ones of Red Hat (question above)

Reply:
@iranzo, I made some additional changes. @ahardin-rh, this is the file we didn't need to worry about since it won't be published.

Reply:
@johnwilkins you mean in this ipi-install-ipv6-network-requirements.adoc or in others? I see that rebase is needed, I can wait and try to apply it on top of upstream docs to get them in sync.

thx